hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -131,8 +131,9 @@ const (
 	controlPlanePKIOperatorSignsCSRsLabel                      = "io.openshift.hypershift.control-plane-pki-operator-signs-csrs"
 	useRestrictedPodSecurityLabel                              = "io.openshift.hypershift.restricted-psa"
 
-	etcdEncKeyPostfix    = "-etcd-encryption-key"
-	managedServiceEnvVar = "MANAGED_SERVICE"
+	etcdEncKeyPostfix                     = "-etcd-encryption-key"
+	managedServiceEnvVar                  = "MANAGED_SERVICE"
+	aroHCPKeyVaultManagedIdentityClientID = "ARO_HCP_KEY_VAULT_USER_CLIENT_ID"
```

Comment:
```suggestion
	aroHCPKeyVaultManagedIdentityClientID = "ARO_HCP_KEYVAULT_USER_CLIENT_ID"
```

Reply:
I am planning to keep them at two words since that's what is in their documentation - https://azure.microsoft.com/en-us/products/key-vault/